### PR TITLE
Lesson2 razor

### PR DESCRIPTION
**Pull request by Gennadievi4**

### CODE REVIEW

File: WebStore/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using WebStore.Services;
+
+namespace WebStore.Controllers
+{
+    public class EmployeesController : Controller
+    {
+
+        private readonly List<Employee> _Db;
+        public EmployeesController(DbInMemory db) => _Db = db._Employees;
+        public IActionResult Index() => View(_Db);
```
Comment (artemfomin) [inline WebStore/Controllers/EmployeesController.cs]: Do not shorten controller methods

File: WebStore/Controllers/EmployeesController.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using System.Linq;
+using WebStore.Models;
+using WebStore.Services;
+
+namespace WebStore.Controllers
+{
+    public class EmployeesController : Controller
+    {
+
+        private readonly List<Employee> _Db;
```
Comment (artemfomin) [inline WebStore/Controllers/EmployeesController.cs]: Trash

File: WebStore/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using WebStore.Models;
+using WebStore.Services;
+
+namespace WebStore.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly List<Employee> _Db;
```
Comment (artemfomin) [inline WebStore/Controllers/HomeController.cs]: Trash

File: WebStore/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        }
+
+        public IActionResult Blogs() => View();
+        public IActionResult BlogSingle() => View();
+        public IActionResult Cart() => View();
+        public IActionResult CheckOut() => View();
+        public IActionResult ContactUs() => View();
+        public IActionResult Login() => View();
+        public IActionResult ProductDetails() => View();
+        public IActionResult Shop() => View();
```
Comment (artemfomin) [inline WebStore/Controllers/HomeController.cs]: Aaaaaaarghhh

File: WebStore/Services/DbInMemory.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using WebStore.Models;
+
+namespace WebStore.Services
+{
+    public class DbInMemory
```
Comment (artemfomin) [inline WebStore/Services/DbInMemory.cs]: It's not a DB

File: WebStore/Startup.cs [hunk 6]
```diff
@@ -12,10 +13,12 @@ public class Startup
         private readonly IConfiguration _Configuration;
         public Startup(IConfiguration Configuration)
         {
-
+            _Configuration = Configuration;
         }
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddMvc().AddRazorRuntimeCompilation();
+            services.AddSingleton<DbInMemory>();
```
Comment (artemfomin) [inline WebStore/Startup.cs]: Why singleton?

File: WebStore/Services/DbInMemory.cs (hunk 5 above)
Comment (Gennadievi4) [reply WebStore/Services/DbInMemory.cs]: This is testing data

File: WebStore/Startup.cs (hunk 6 above)
Comment (Gennadievi4) [reply WebStore/Startup.cs]: Because the object will not change its state

**Pull request merged**